### ISSUE TITLE
Stopped working around n8n v1.63.4

### ISSUE THREAD

**Issue by rthomas6charter** [flipswitchingmonkey/n8nworkerview#3]
This utility seems to have stopped working somewhere between n8n version 1.61.0 and 1.63.4.  Not sure if n8n has changed how it uses Redis, or if the channel names are just different now, but the display no longer shows any workers or tasks.  If I find a specific cause and/or workaround, I'll post a response here.

**Comment by flipswitchingmonkey** [comment]
Yes they changed it so the message packets which the tool is reading are only available in the enterprise license now (plus presumable the format was adjusted sure, I'm not certain about that)

**Comment by rthomas6charter** [comment]
Did some poking around and found that it does not seem to be related to licensing.
Older version of n8n responds to:
`{"senderId":"insight","command":"getStatus","payload":"","targets":null}`
Newer version responds to:
`{"senderId":"insight","command":"get-worker-status","payload":"","targets":null}`
Time permitting, I'll fork and submit a PR.
